priv/repo/migrations/20161212005641_create_stripe_file_upload.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateStripeFileUpload do
+  use Ecto.Migration
+
+  def change do
+    create table(:stripe_file_upload) do
+      add :created, :datetime
+      add :id_from_stripe, :string, null: false
+      add :purpose, :string
+      add :size, :integer
+      add :type, :string
+      add :url, :string
+    end
```

Comment:
yeah I guess that is where I was confused b/c Would it look like this? `add :stripe_connect_account_id, references(:stripe_connect_accounts)` or `add :stripe_connect_account_id, references(:stripe_connect_accounts, on_delete: nothing)`